plugins/hub/osbuild.py

```diff
@@ -95,6 +190,14 @@
                     "type": "object",
                     "$ref": "#/definitions/ostree"
                 },
+                "upload_options": {
+                    "oneOf": [
+                        {"$ref": "#/definitions/AWSEC2UploadOptions"},
+                        {"$ref": "#/definitions/AWSS3UploadOptions"},
+                        {"$ref": "#/definitions/GCPUploadOptions"},
+                        {"$ref": "#/definitions/AzureUploadOptions"}
+                    ]
```

Comment:
I would just route that through and let Composer verify the schema, we do the same for `customizations`. We are not the source of truth here and it will be tedious to keep them in sync.

Reply:
ACK

Reply:
removed....